assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -225,18 +226,19 @@ const baseActions = {
 	 * @since n.e.x.t Introduced the ability to explictly define a module store name.
 	 *
-	 * @param {string}      slug                             Module slug.
-	 * @param {Object}      [settings]                       Optional. Module settings.
-	 * @param {string}      [settings.storeName]             Optional. Module storeName. If none is provided we assume no store exists for this module.
-	 * @param {string}      [settings.name]                  Optional. Module name. Default is the slug.
-	 * @param {string}      [settings.description]           Optional. Module description. Default empty string.
-	 * @param {WPComponent} [settings.Icon]                  Optional. React component to render module icon. Default none.
-	 * @param {number}      [settings.order]                 Optional. Numeric indicator for module order. Default 10.
-	 * @param {string}      [settings.homepage]              Optional. Module homepage URL. Default empty string.
-	 * @param {WPComponent} [settings.SettingsEditComponent] Optional. React component to render the settings edit panel. Default none.
-	 * @param {WPComponent} [settings.SettingsViewComponent] Optional. React component to render the settings view panel. Default none.
-	 * @param {WPComponent} [settings.SetupComponent]        Optional. React component to render the setup panel. Default none.
-	 * @param {Function}    [settings.checkRequirements]     Optional. Function to check requirements for the module. Throws a WP error object for error or returns on success.
+	 * @param {string}      slug                                        Module slug.
+	 * @param {Object}      [settings]                                  Optional. Module settings.
+	 * @param {string}      [settings.storeName]                        Optional. Module storeName. If none is provided we assume no store exists for this module.
+	 * @param {string}      [settings.name]                             Optional. Module name. Default is the slug.
+	 * @param {string}      [settings.description]                      Optional. Module description. Default empty string.
+	 * @param {WPComponent} [settings.Icon]                             Optional. React component to render module icon. Default none.
+	 * @param {number}      [settings.order]                            Optional. Numeric indicator for module order. Default 10.
+	 * @param {string}      [settings.homepage]                         Optional. Module homepage URL. Default empty string.
+	 * @param {WPComponent} [settings.SettingsEditComponent]            Optional. React component to render the settings edit panel. Default none.
+	 * @param {WPComponent} [settings.SettingsViewComponent]            Optional. React component to render the settings view panel. Default none.
+	 * @param {WPComponent} [settings.SettingsSetupIncompleteComponent] Optional. React component to render the incomplete settings panel. Default DefaultSettingsSetupIncomplete.
```

Comment:
Sorry, I think I was incorrect here which may have been confusing as to where this goes. The default here should remain "none" as it was, as there is no default _parameter value_ – the default is only applied when the module is retrieved, not on registration.